bravo/src/main/java/com/king/bravo/reader/KeyedStateReader.java

```diff
@@ -35,15 +34,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.king.bravo.types.KeyedStateRow;
-import com.king.bravo.utils.StateMetadataUtils;
+import java.io.IOException;
+import java.util.List;
 
 public abstract class KeyedStateReader<K, V, O> extends RichFlatMapFunction<KeyedStateRow, O>
 		implements ResultTypeQueryable<O> {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(KeyedStateReader.class);
 	private static final long serialVersionUID = 1L;
 
+	// The name "window-contents" appears as a plain string in many places in Flink source code. There is no constant
+	// that we could refer from Flink.
+	private static final String REDUCER_STATE_NAME = "window-contents";
```

Comment:
@gyfora, now that I'm looking at this, the correct term to use would be "window state" instead of "reducer state". Because it's not specific to reducer type of window operator. Is it ok if I make a PR to adjust the terminology?

Reply:
sounds good!